test/util/server/server.go

```diff
@@ -356,17 +358,11 @@ func StartConfiguredMasterWithOptions(masterConfig *configapi.MasterConfig) (str
 		return "", err
 	}
 
-	/********** controllers the old way **********/
-
-	// controllers need new address they can bind debug and healthz to
-	controllersAddrStr, err := FindAvailableBindAddress(10000, 29999)
-	if err != nil {
-		return "", fmt.Errorf("Couldn't find free address for controllers: %v", err)
+	if err := startKubernetesControllers(masterConfig, adminKubeConfigFile); err != nil {
+		return "", err
 	}
-	masterConfig.ServingInfo.BindAddress = controllersAddrStr
```

Comment:
@deads2k ha, this switch uncovered an error, that we're serving more ciphers in the API that we should. The `TestMasterTLSDefaults` test before was working against controller address and that's why it passed, but should fail :man_facepalming: another :wilted_flower: